geopandas/base.py

```diff
@@ -492,6 +492,61 @@ def centroid(self):
         """
         return _delegate_property("centroid", self)
 
+    def concave_hull(self, ratio=0.0, allow_holes=False):
+        """Returns a ``GeoSeries`` of geometries representing the concave hull
+        of each geometry.
+
+        The concave hull of a geometry is the smallest concave `Polygon`
+        containing all the points in each geometry, unless the number of points
+        in the geometric object is less than three. For two points, the concave
+        hull collapses to a `LineString`; for 1, a `Point`.
```

Comment:
It would be good to have an indication of how the algorithm actually works (there are more than one method out there) but given neither shapely specifies that, it is of a low priority. We would have to dig into GEOS or JTS.

Reply:
I've put some details in which I lifted from GEOS, and added a link to the GEOS docs